src/main/scala/cromwell/engine/callactor/FinalCallActor.scala

```diff
@@ -0,0 +1,14 @@
+package cromwell.engine.callactor
+
+import akka.actor.ActorRef
+import cromwell.engine.WorkflowDescriptor
+import cromwell.engine.callexecution.CallExecutionActor
```

Comment:
maybe the package structure should be like `cromwell.engine.call` and under that you can have `execution`, `backend`, `final`, etc